app/pb_kits/playbook/pb_date_picker/sass_partials/_overrides.scss

```diff
@@ -0,0 +1,37 @@
+// media query non mobile styles
```

Comment:
Lots of hardcoded widths here. Any concerns with resizing?

Reply:
Widths were added to accommodate extra padding.  Wanted to remove padding on smaller screens to maintain mobile usability.  Works as expected on any size screen.